package.json

```diff
@@ -5,8 +5,9 @@
     "type-check": "tsc",
     "generate": "npm-run-all generate:*",
     "generate:payment-ecommerce-v1": "rimraf generated/definitions/payment-ecommerce && shx mkdir -p generated/definitions/payment-ecommerce && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/v1.632.1/src/domains/ecommerce-app/api/ecommerce-checkout/v1/_openapi.json.tpl --no-strict --out-dir ./generated/definitions/payment-ecommerce --request-types --response-decoders --client",
-    "generate:payment-ecommerce-v2": "rimraf generated/definitions/payment-ecommerce-v2 && shx mkdir -p generated/definitions/payment-ecommerce-v2 && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/v1.632.1/src/domains/ecommerce-app/api/ecommerce-checkout/v2/_openapi.json.tpl --no-strict --out-dir ./generated/definitions/payment-ecommerce-v2 --request-types --response-decoders --client",
+    "generate:payment-ecommerce-v2": "rimraf generated/definitions/payment-ecommerce-v2 && shx mkdir -p generated/definitions/payment-ecommerce-v2 && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/CHK-4435-add-pay-handler-apim-policies/src/domains/ecommerce-app/api/ecommerce-checkout/v2/_openapi.json.tpl --no-strict --out-dir ./generated/definitions/payment-ecommerce-v2 --request-types --response-decoders --client",
```

Comment:
TODO to change branch with tag ref after dependent pr merge for all payment-ecommerce-v* generate instances